### DISCUSSION TITLE
Firefly Wallet doesn’t show IOTA or Staking.

### DISCUSSION THREAD

**Discussion by Abduu75** [iotaledger/firefly#5388] [Help & Support]
Hey guys,

i have a problem wich I could not fixed it yet. I already asked on the discord and nobody could help me.

My problem is:
I got a new MacBook, downloaded the firefly wallet and put the 24 word recovery phrases in and logged into my account. But now I don’t see any funds like nothing 0 iotas no staking nothing…

things I done but did not helped yet:

- clicked over 100 times on the balance finder - no funds found.
- Changed the node to manual and than back to Auto again
- Deleted the wallet and made a new one with the same 24 word recovery phrase

I don’t have access to my old macbook anymore…

I also check the tangle explorer and my iotas are still there on my old adress - my wallet right now is showing me a new one.

Can anyone please help me did someone had same problem or having also the same problem. Maybe wait on a new firefly update?

**Comment by MiKeF05** [comment]
Either you have the wrong 24 words or your macbook blocks the requests to the node.

**Reply by Abduu75** [reply]
What can I do that it’s not blocking the request to the node?
